Makefile.cflags

```diff
@@ -51,6 +51,9 @@ endif
 # Forbid common symbols to prevent accidental aliasing.
 CFLAGS += -fno-common
 
+# Enable all default warnings
+CFLAGS += -Wall
```

Comment:
Enabling the warnings does not turn them into errors. `-Werror` is only enabled in `make buildtest`